framework-support/jobrunr-quarkus-extension/quarkus-jobrunr/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -8,10 +9,11 @@ metadata:
     - "cron"
     - "distributed background jobs"
   guide: "https://www.jobrunr.io/en/documentation/configuration/quarkus/"
+  icon-url: "https://raw.githubusercontent.com/jobrunr/website/master/static/jobrunr-logo.png"
+  sponsor: "Rosoco BV"
   categories:
     - "miscellaneous"
   status: "preview"
```

Comment:
Let's remove status preview. We know it is being used in PRD for a long time.

Reply:
I set it to `stable`